Marlin/src/lcd/ultralcd.h

```diff
@@ -27,7 +27,7 @@
   #include "../libs/buzzer.h"
 #endif
 
-#define HAS_ENCODER_ACTION (HAS_LCD_MENU || ENABLED(ULTIPANEL_FEEDMULTIPLY))
+#define HAS_ENCODER_ACTION (HAS_LCD_MENU)
```

Comment:
The purpose of this conditional is to indicate that there's something in the LCD that can make use of the encoder knob. That includes: menu navigation and value editing, and changing the feed multiply value on the status screen, which is available even when `NO_LCD_MENUS` is enabled.

Reply:
Okay, but if the code remains snippets, then compiled does not work.

```c++
In function `MarlinUI::update_buttons()':
<artificial>:(.text+0x138): undefined reference to `MarlinUI::encoderDirection'
<artificial>:(.text+0x144): undefined reference to `MarlinUI::encoderDirection'
In function `MarlinUI::update()':
<artificial>:(.text+0x5d2): undefined reference to `MarlinUI::encoderDirection'
```